website/docs/development.md

```diff
@@ -47,7 +47,7 @@ created run configuration to your needs, e.g. by adding an argument and options
 ## Testing
 
 ORT uses [Kotest](https://github.com/kotest/kotest) as the test framework. For running tests and individual test cases
-from the IDE, the [Kotest plugin](https://plugins.jetbrains.com/plugin/14080-kotest) needs to be installed. Afterwards,
+from the IDE, the [Kotest plugin](https://plugins.jetbrains.com/plugin/14080-kotest) needs to be installed. Afterward,
```

Comment:
Note that while "afterward" seems to be exclusively American English, "afterwards" is not exclusively British English, but also American English. Personally, I prefer "afterwards" over "afterward".

Reply:
From my research, I found that in AE the term "afterward" is preferred. To my understanding we stick to AE. Given that, it implies to me that "afterward" makes sense. 

Do you disagree with this argumentation? If so why?

Reply:
It's correct that we should stick to AE, and I'd agree with the change if proof was provided that AE prefers "afterward" over "afterwards". So, please share the reference you found during your research.